scripts/variantstore/wdl/ImportGenomes.wdl

```diff
@@ -65,16 +70,28 @@ workflow ImportGenomes {
       superpartitioned = "true",
       partitioned = "true",
       uuid = "",
+      service_account_json = service_account_json,
       preemptible_tries = preemptible_tries,
       docker = docker_final
   }
 
   scatter (i in range(length(input_vcfs))) {
+    if (defined(service_account_json)) {
```

Comment:
Could this copy either (a) just be embedded into the CreateImportTSVs task or (b) skip localizing entirely by having the GATK stream the data source bucket with the right credentials?    The way I read this, if we are importing 1000 files we spin up 1000 VMs just to copy the files and then 1000 more VMs to transform that file to TSV?